test/BodhiToken_spec.js

```diff
@@ -29,9 +36,13 @@ contract('BodhiToken', function(accounts) {
       let expectedSaleAmount = web3.toBigNumber(web3.toWei(60e6, "ether"));
       assert.equal(maxTokenForSale.toString(), expectedSaleAmount.toString(), "Sale amount does not match.");
 
-      let totalSupply = web3.toBigNumber(await token.tokenTotalSupply());
-      let expectedTotalSupply = web3.toBigNumber(web3.toWei(100e6, "ether"));
-      assert.equal(totalSupply.toString(), expectedTotalSupply.toString(), "Total token supply does not match.");
+      let totalTokenSupply = web3.toBigNumber(await token.tokenTotalSupply());
+      let expectedTotalTokenSupply = web3.toBigNumber(web3.toWei(100e6, "ether"));
+      assert.equal(totalTokenSupply.toString(), expectedTotalTokenSupply.toString(), "Total token supply does not match.");
```

Comment:
why compare after casting to str?

Reply:
the JS tests do not like to compare big numbers (they just fail for some reason). and Strings can handle any amount of number values (length).